opennms-doc/guide-install/src/asciidoc/text/minion/configure-minion.adoc

```diff
@@ -30,6 +30,15 @@ admin@minion()> scv:set opennms.http minion minion
 admin@minion()> scv:set opennms.broker minion minion
 ----
 
+Another way to configure credentials is to use the `scvcli` utility in your _Minion_ `bin` directory.
+
+.Configure the credentials by using the command line utility `scvcli`
+[source]
+----
+[root@localhost /usr/share/minion]# $ ./bin/scvcli set opennms.http minion minion
```

Comment:
The Debian packages don't create `/opt/minion`, so this won't work with the Debian packages now.

Maybe we should append something like:
```
NOTE: If you are using the Debian packages, use `/usr/share/minion` instead of `/opt/minion`
```

Sorry for the confusion :)